### PR TITLE
feat(cli): use pwd as default value for `--tfdir`

### PR DESCRIPTION
**Pull request by hakamadare**

### CODE REVIEW

File: cmd/infracost/main.go [hunk 1]
```diff
@@ -41,9 +41,11 @@ func main() {
 				TakesFile: true,
 			},
 			&cli.StringFlag{
-				Name:      "tfdir",
-				Usage:     "Path to the Terraform project directory",
-				TakesFile: true,
+				Name:        "tfdir",
+				Usage:       "Path to the Terraform project directory",
+				TakesFile:   true,
+				Value:       get_cwd(),
```
Comment (aliscott) [inline cmd/infracost/main.go:47]:
```suggestion
				Value:       getcwd(),
```

File: cmd/infracost/main.go [hunk 2]
```diff
@@ -150,3 +152,13 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func get_cwd() string {
```
Comment (aliscott) [inline cmd/infracost/main.go:156]:
```suggestion
func getcwd() string {
```

File: cmd/infracost/main.go [hunk 3]
```diff
@@ -150,3 +152,13 @@ func main() {
...
+	cwd, err := os.Getwd()
+	if err != nil {
+		log.Warn(err)
+		cwd = "UNKNOWN"
```
Comment (aliscott) [inline cmd/infracost/main.go:160]:
```suggestion
		cwd = ""
```
Any benefit of having "UNKNOWN" over just a blank string? We check for a blank string when validating the args here: https://github.com/infracost/infracost/blob/master/internal/providers/terraform/provider.go#L23

**Review by aliscott — COMMENTED** [review]
Thanks @hakamadare! This is a good idea.

File: cmd/infracost/main.go (hunk 3 above)
Comment (hakamadare) [reply cmd/infracost/main.go:160]: not particularly; i don't tend to like blank strings b/c i like to interpolate values into log messages, and blank strings don't interpolate as helpfully as placeholders like "UNKNOWN", but this is a weak preference.

**Comment by aliscott** [comment]
Thanks @hakamadare!

**Pull request merged**